docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-space-syntax.md

```diff
@@ -10,13 +10,19 @@ space_option:
 
 This statement creates a new space with the given name. SPACE is a region that provides physically isolated graphs in **Nebula Graph**. An error occurs if the database exists.
 
+## IF NOT EXIST
+
+We added `If NOT EXIST` keyword for creating spaces. This keyword automatically detects if the corresponding space exists. If it does not exist, a new one is created. Otherwise, no space is created.
```

Comment:
We added `If NOT EXIST` keyword -> We added the `If NOT EXIST` keywords

Please change others